platforms/ide/tooling-api/src/main/java/org/gradle/tooling/internal/consumer/DefaultFileComparisonTestAssertionFailure.java

```diff
@@ -29,7 +29,15 @@ public class DefaultFileComparisonTestAssertionFailure extends DefaultTestAssert
     private final byte[] expectedContent;
     private final byte[] actualContent;
 
-    public DefaultFileComparisonTestAssertionFailure(String message, String description, String expected, String actual, List<? extends Failure> causes, String className, String stacktrace, byte[] expectedContent, byte[] actualContent) {
+    public DefaultFileComparisonTestAssertionFailure(@Nullable String message,
```

Comment:
❌ This change is unrelated to the PR and actually breaks the formatting also used in `DefaultTestAssertionFailure`, `DefaultTestFrameworkFailure`.